R/queue.R

```diff
@@ -22,7 +22,8 @@ Queue <- R6::R6Class(
       message(t_("QUEUE_STARTING"))
       queue_id <- hintr_queue_id(queue_id)
       self$queue <- rrq::rrq_controller(queue_id, con)
-      self$queue$worker_config_save("localhost", heartbeat_period = 3)
+      self$queue$worker_config_save("localhost", heartbeat_period = 3,
+                                    queue = c("calibrate", "run"))
```

Comment:
Do we have a string constant set anywhere? Replacing these string literals with `queue_calibrate` and `queue_run` (and `queues <- c(queue_calibrate, queue_run)`) might help with some typos

You can also use the default queue for run, which might be simpler?